src/Env/index.ts

```diff
@@ -96,9 +98,15 @@ export class Env implements EnvContract {
 		/**
 		 * Perform validations by reading the environment variables
 		 */
+		const errors: string[] = []
 		Object.keys(this.validationSchema).forEach((key) => {
-			this.envCache[key] = this.validationSchema[key](key, this.get(key))
+			try {
+				this.envCache[key] = this.validationSchema[key](key, this.get(key))
+			} catch (err) {
+				errors.push((err as Exception).message)
+			}
 		})
+		if (errors.length > 0) throw new Exception(errors.join(EOL), 500)
```

Comment:
```suggestion
		if (errors.length > 0) throw new Exception(errors.join('\n'), 500)
```

I don't think anyone would need the separator to be `\r\n` on Windows. If you take `error.stack` for example, V8 always sets it to `\n`.